composer.json

```diff
@@ -18,6 +18,8 @@
     ],
     "require": {
         "php": ">=7.3",
+        "friendsofphp/php-cs-fixer": "v2.18.6",
+        "pedrotroller/php-cs-custom-fixer": "v2.24.1",
```

Comment:
```suggestion
        "pedrotroller/php-cs-custom-fixer": "~2.24.0",
```

Reply:
No, the goal is to avoid wild upgrade of `friendsofphp/php-cs-fixer` or `pedrotroller/php-cs-custom-fixer`.